blackbox-tests/generate-certificates.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu
+
+openssl genrsa -out ca-key.pem 4096
+openssl req -new -x509 -days 365 -key ca-key.pem -sha256 -out ca.pem \
+  -subj "/C=GB/ST=London/L=London/O=Global Security/OU=IT Department/CN=example.com"
```

Comment:
I mean yes it's arbitrary, but London/UK?

Reply:
lol